lib/getters/user.ts

```diff
@@ -4,10 +4,22 @@ import type { CustomerAccount } from "@/server/types/GraphQL"
 export const isLoggedInUser = (user: CustomerAccount) => {
   return user?.isAnonymous === false
 }
+export const getUserEmail = (user: CustomerAccount) => {
+  return user?.emailAddress
+}
+export const getUserFirstName = (user: CustomerAccount) => {
+  return user?.firstName
+}
+export const getUserLastName = (user: CustomerAccount) => {
+  return user?.lastName
+}
```

Comment:
Make all above statement single one liner
ex. export const getUserLastName = (user: CustomerAccount) => user?.lastName

Reply:
No longer in use